src/ec_date.erl

```diff
@@ -709,11 +709,7 @@ pad6(X) when is_integer(X) ->
 ltoi(X) ->
     list_to_integer(X).
 
--ifdef(unicode_str).
 uppercase(Str) -> string:uppercase(Str).
```

Comment:
should we just keep the intermediary call here or also fold in the `uppercase/1` call to be `string:uppercase/1`?

Reply:
Folded all down to `string:uppercase/1`: no blocker.